### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by AlecMcCutcheon** [Tadashi-Hikari/Athena#5]
Just wondering

**Comment by hikari-tadashi** [comment]
Yeah. I've just been doing redesigns on my local computer, rather than under the git account. Currently, working with Vosk, PocketSphinx, and DeepSpeech to create a better out-of-the-box STT system

**Comment by andsofine** [comment]
any updates?

**Comment by thomasgroch** [comment]
Hi everyone,
Has there been any progress since the last update?
I’m really curious to know if the improvements have moved forward.
Also, is there a website or page with more information about the project?

Thanks, and great work so far!
